db/schema.rb

```diff
@@ -124,6 +124,31 @@
     t.integer  "round",                    default: 1
   end
 
+  create_table "events", force: :cascade do |t|
```

Comment:
I'm not sure what this and the `job_offers` table are doing here. I think it might have to do with the problems you were having running the migrations — when you set it up from the db dump — but I really don't know what the best way to remove these without messing with the schema is.. 🙈